.gitignore

```diff
@@ -183,10 +183,6 @@ tags
 
 ### VisualStudioCode ###
 .vscode/*
-!.vscode/settings.json
```

Comment:
Why is this related?

Reply:
Because I believe that these exceptions are not needed on the server

Reply:
Doesn't sound related to adding a skeleton. It'll still work without this. So it doesn't belong to this PR.

Reply:
Should I make squash with the previous commit?

Reply:
It doesn't belong to the PR, nor to the commit. So no. Just move it out.

Reply:
rebase -i
delete commit?

Reply:
You can cherry-pick it to a separate branch if you still want to apply it. Just don't keep it in the branch bound to this PR.

Reply:
I think not need to change gitignote in this pr. If you want to change gitignote, need create a new pr

Reply:
thanks